src/pytsql/grammar/sa_tsql.py

```diff
@@ -142,6 +142,12 @@ def syntaxError(self, recognizer, offendingSymbol, line, column, msg, e):
 
 
 def _py_parse(stream:InputStream, entry_rule_name:str, sa_err_listener:SA_ErrorListener=None) -> ParseTree:
+    warnings.warn(
```

Comment:
As you pointed out earlier - this is an auto generated file, so anyone would have to retype this warning if they're updating the grammar, right? Maybe we should just import `USE_CPP_IMPLEMENTATION` inside `tsql.py` and do the warning there in case it's false. Do you see any downsides to that?